rspace/src/test/scala/coop/rchain/rspace/ReplayRSpaceTests.scala

```diff
@@ -275,18 +277,20 @@ trait ReplayRSpaceTests extends ReplayRSpaceTestsBase[String, Pattern, String, S
                            false,
                            peeks = SortedSet(0)
                          )
-        replayProduce1 <- replaySpace.produce(channels(1), datum, false)
-        replayProduce2 <- replaySpace.produce(channels(0), datum, false)
+        replayProduce1  <- replaySpace.produce(channels(1), datum, false)
+        replayProduce2  <- replaySpace.produce(channels(0), datum, false)
+        resultProduce2a <- replaySpace.produce(channels(0), datum, false)
```

Comment:
wow, I wasn't even aware that you can do that in a single `for`:

```scala
for {
x <- Task.unit
x <- Task.unit
} yield ()
```

No surprise given the desugaring, but I'd expect some linter to yell at this :joy: